openms-wiki/building-openms.md

```diff
@@ -0,0 +1,11 @@
+# Build OpenMS
+
+To build OpenMS, please follow the build instructions:
+
+* [Instructions for Linux](instructions-for-linux.md)
+* [Instructions for OSX](instructions-for-osx.md)
+* [Instructions for Windows](instructions-for-windows.md)
+
+For certain special builds, find instructions here:
```

Comment:
let's say:

Instructions to build pyOpenMS, can be found [here]().

Reply:
https://ux.stackexchange.com/questions/12100/why-shouldnt-we-use-words-such-as-here-and-this-in-textlinks